doc/user/INSTALLATION.md

```diff
@@ -1,20 +1,43 @@
 # Installation
 
-You can find the binary release packages for Windows, Linux and macOS on the [releases page](https://github.com/f3d-app/f3d/releases).
+## Current release packages (1.3.1)
+
+| Platform | Files |
+| -------- | ----- |
+| Windows Installer | [f3d-1.3.1-win64.exe](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-win64.exe) |
+| Windows (Portable) | [f3d-1.3.1-win64.zip](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-win64.zip) |
+| MacOS Installer | [f3d-1.3.1-Darwin.dmg](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-Darwin.dmg) |
+| Debian-based Linux Installer | [f3d-1.3.1-Linux.deb](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-Linux.deb) |
+| Linux (Portable) | [f3d-1.3.1-Linux.tar.xz](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-Linux.tar.xz) |
+
+## Alternatives
+
+You can find the older binary release packages for Windows, Linux and macOS on the [releases page](https://github.com/f3d-app/f3d/releases).
 See the [desktop integration](DESKTOP_INTEGRATION.md) section in order actually integrate the binary release in your desktop.
 Alternatively, you can build F3D yourself by following the [build](dev/BUILD.md) guide.
 
 You can also find packages for the following package managers:
 
+### Windows
+
+- [Scoop](https://scoop.sh/#/apps?q=f3d&s=0&d=1&o=true)
+
+### MacOS
+
+- [Homebrew](https://formulae.brew.sh/formula/f3d)
+
+### Linux
+
 - [Ubuntu](https://packages.ubuntu.com/search?keywords=f3d&searchon=names&exact=1&suite=all&section=all)
 - [Debian](https://packages.debian.org/search?keywords=f3d&searchon=names&exact=1&suite=all&section=all)
 - [Arch User Repository](https://aur.archlinux.org/packages/f3d)
-- [FreeBSD](https://cgit.freebsd.org/ports/tree/graphics/f3d)
 - [NixOS](https://search.nixos.org/packages?channel=22.05&show=f3d&from=0&size=50&sort=relevance&type=packages&query=f3d)
 - [openSUSE](https://software.opensuse.org/package/f3d)
 - Fedora and others through [openSUSE OBS](https://build.opensuse.org/package/show/home:AndnoVember:F3D/f3d)
-- [MacOS Homebrew](https://formulae.brew.sh/formula/f3d)
-- [Windows Scoop](https://scoop.sh/#/apps?q=f3d&s=0&d=1&o=true)
+- [Guix](https://guix.gnu.org/en/packages/f3d-1.3.1/)
+
+### Other
```

Comment:
Not sure if Flatpack and Spack should be under Other or Linux, what do you think @mwestphal ?

Reply:
Flatpack is linux specific, spack is not (supports linux, macos and strange HPC stuff).